conf/test_one_sample.config

```diff
@@ -50,3 +50,9 @@ params {
     vep_filters          = "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/hgnc.txt"
     vep_cache_version    = 107
 }
+
+process {
+    withName: '.*FILTERVEP.*' {
+        container  = "docker.io/ensemblorg/ensembl-vep:release_107.0"
```

Comment:
Just curious to know why you don't update VEP to 110 in the test?

Reply:
I was getting an error about lack of space on device from github CI.. Then, I noticed that the image of vep v110 we use on nf-core modules is about 1.1GB is size, and the one I was using locally had a size of 300MB. So I made the switch to see if that solved CI error, and it looks like it did. 😄